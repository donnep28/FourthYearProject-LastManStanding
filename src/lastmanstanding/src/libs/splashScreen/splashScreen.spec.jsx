import React from 'react';
import { shallow } from 'enzyme';
import SplashScreen from './splashScreen';

describe('Splash Screen', () => {
    describe('Snapshot', () => {
        it('renders', () => {
          const component = shallow(
            <SplashScreen/>,
          );
          expect(component).toMatchSnapshot();
        });
    });
});