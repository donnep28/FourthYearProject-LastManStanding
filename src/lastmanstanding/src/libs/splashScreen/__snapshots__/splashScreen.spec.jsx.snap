// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash Screen Snapshot renders 1`] = `
<div
  className="splashScreen"
  style={
    Object {
      "background": "#ffffff",
    }
  }
>
  <img
    alt="splash screen logo"
    className="splashScreenImage"
    src="LMS2.gif"
  />
</div>
`;
